src/submission/submission-to-clinical/merge-submission.ts

```diff
@@ -1,17 +1,27 @@
 import { DeepReadonly } from 'deep-freeze';
-import { Donor, Treatment, ClinicalInfo, FollowUp } from '../../clinical/clinical-entities';
+import {
```

Comment:
it looks like a lot of refactoring in this file, but its mostly just moving things around to make things more consistent (and hopefully easier to follow). The main thing being added here is that each `update` function now returns the clinicalEntity it updated, so that we can update the stats in that clinicalEntity and the aggregatedStats in the donor.

Reply:
Refactoring is a good thing